include/circt/Dialect/FIRRTL/Passes.td

```diff
@@ -81,6 +81,20 @@ def RemoveUnusedPorts : Pass<"firrtl-remove-unused-ports", "firrtl::CircuitOp">
   ];
 }
 
+def IMDCE : Pass<"firrtl-imdce", "firrtl::CircuitOp"> {
```

Comment:
Maybe stupid but the other pass is `IMConstProp`, if we follow the convention this should be `IMDeadCodeElim`